src/main/start-main-application/lens-window/application-window/application-window.injectable.ts

```diff
@@ -50,7 +50,7 @@ const applicationWindowInjectable = getInjectable({
           ipcMain.once(bundledExtensionsLoaded, () => resolve());
         });
 
-        await viewHasLoaded;
+        await Promise.race([viewHasLoaded, new Promise(resolve => setTimeout(resolve, 20_000))]);
```

Comment:
No behavioral unit test for this new feature?

Reply:
This alarmingly odd code should explain why it is required.

Reply:
oh sorry, I should have removed this. This was part of testing because I couldn't get the window to open.